en/full-node.md

```diff
@@ -1107,10 +1107,47 @@ automatically started minimized in the task bar.
 #### Bitcoin Core Daemon {#osx-daemon}
 {:.no_toc}
 
-If you can provide instructions and screenshots for running the latest
-version of Bitcoin Core daemon on OS X Yosemite, please [open an
-issue](https://github.com/bitcoin-dot-org/bitcoin.org/issues/new) and we'll tell
-you what we need.
+The Bitcoin Core daemon (bitcoind) is not included in the .dmg file you may have downloaded to install Bitcoin-QT. Bitcoind, along with its support binaries, is instead included in the OS X .tar.gz file listed on the official Bitcoin Core download page. To download this file using Terminal, execute the following command:
+
+    curl -O https://bitcoin.org/bin/bitcoin-core-0.11.0/bitcoin-0.11.0-osx64.tar.gz
+
+{{verifyReleaseSignatures}}
+
+Extract bitcoind and its support binaries from the archive we just downloaded by running this command in Terminal:
+
+    tar -zxvf bitcoin-0.11.0-osx64.tar.gz
```

Comment:
Yes, `{{site.DOWNLOAD_VERSION}}` should produce the current version number only (current output: 0.11.0)

The Mac filenames used to change fairly often, but I think we've sorted that out and they've been stable for the last few releases.